test/cqlpy/test_filtering.py

```diff
@@ -486,3 +487,17 @@ def test_selecting_a_regular_column_does_not_poison_filtering_of_a_static_row(cq
         cql.execute(f"INSERT INTO {table} (a, s) VALUES (1, 2)")
         res = cql.execute(f"SELECT a, b, c, s FROM {table} WHERE s = 2 ALLOW FILTERING")
         assert list(res) == [(1, None, None, 2)]
+
+# Test that some other CQL commands support the same filtering as SELECT, i.e. one based on the WHERE clause.
+# Note we don't need to explicitly assert the results, as we know that WHERE clause works correctly from the tests above,
+# so we're just testing CQL interface: if these commands were not allowed, the test would fail executing invalid query.
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #18066")]), "vnodes"],
+                         indirect=True)
+def test_filtering_operations(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, 'p int, c int, s int, PRIMARY KEY (p, c)') as table:
```

Comment:
nitpick: when I saw "s" I immediately thought - "static column". Which it isn't. Maybe something else, like "x" or "a", will be better.